public/app/plugins/datasource/loki/components/LokiExploreQueryEditor.tsx

```diff
@@ -25,6 +25,7 @@ export function LokiExploreQueryEditor(props: Props) {
       range={range}
       ExtraFieldElement={
         <LokiOptionFields
+          queryDirection={query.direction ?? datasource.direction ?? 'BACKWARD'}
```

Comment:
I think `queryDirection={query.direction ?? datasource.direction}` should be sufficient as we are always setting direction in datasource and already defaulting to `BACKWARD` there.

Reply:
Nice catch.